### PR TITLE
Add author_authoritative to drive author_browse

### PR DESCRIPTION
**Pull request by billdueber**
Adds new `author_authoritative` to drive browse.

Right now, the definition of `author` is very close to this, The subfields used in the new `to_field` are as dictated to me by Leigh Billings,  and I'm assuming (remembering?) that the selection of subfields in the current code was relatively arbitrary. 

We should pick one of the following:
  * merge this in
  * merge this in and later change the def of `author`
  * just change the def of `author`

### CODE REVIEW

**Comment by billdueber** [comment]
The only subfield we'd be taking away is the 111/711 ‡b, and `b` isn't even defined for that field (leading me to strongly believe that what we have there is totally guesses by me 12 years ago). I'm voting for change the definition of `author`.

**Comment by billdueber** [comment]
Re-jiggered to use recommended subfields to `author` and get them into `author_authority` via a `copyField`

**Comment by niquerio** [comment]
I got it to run locally without issue, so it is fine with me.

Let's wait until Monday to merge it and we can run a full reindex on it then.

**Review by niquerio — APPROVED** [review]
Ran it locally and it indexed fine. I want to wait until Monday to deploy it.

File: umich_catalog_indexing/indexers/common.rb
Comment (billdueber) [reply umich_catalog_indexing/indexers/common.rb]: It shouldn't affect either. For display purposes they want a more expanded set of subfields.  We're concerned mostly with searching and with making sure we're able to match on what the LC puts in the Name Authority File. Our definition of the main author hasn't changed at all, and never matched what they do anyway.

**Pull request merged**
